tests/unit/Statement/Grouper/NullStatementGrouperTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Wikibase\DataModel\Services\Tests\Statement\Grouper;
+
+use PHPUnit_Framework_TestCase;
+use Wikibase\DataModel\Services\Statement\Grouper\NullStatementGrouper;
+use Wikibase\DataModel\Snak\PropertyNoValueSnak;
+use Wikibase\DataModel\Statement\StatementList;
+
+/**
+ * @covers Wikibase\DataModel\Services\Statement\Grouper\NullStatementGrouper
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class NullStatementGrouperTest extends PHPUnit_Framework_TestCase {
+
+	public function test() {
```

Comment:
PHPUnit runs this. I usually start with `test()` and name it when it becomes obvious what it tests.

This is a mistake. I fixed it and a similar one in the other grouper.